src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-your-applications-kubernetes.mdx

```diff
@@ -30,13 +31,19 @@ Before linking Kubernetes metadata to your APM agents, make sure you meet the fo
 
 ### Kubernetes requirements [#kubernetes-req]
 
-To link your applications and Kubernetes, your cluster must have the `MutatingAdmissionWebhook` controller enabled, which requires **Kubernetes 1.9 or higher**.
+To link your applications and Kubernetes, you must be running **Kubernetes 1.16 or higher** and be able to deploy `MutatingWebhookConfiguration`.
```

Comment:
Same as above, we have a compatibility page, should we keep everything there? Otherwise it gets outdated. and This is an example 😄